go/cmd/dolt/commands/tblcmds/export.go

```diff
@@ -193,6 +191,7 @@ func (cmd ExportCmd) ArgParser() *argparser.ArgParser {
 	ap.ArgListHelp = append(ap.ArgListHelp, [2]string{"table", "The table being exported."})
 	ap.ArgListHelp = append(ap.ArgListHelp, [2]string{"file", "The file being output to."})
 	ap.SupportsFlag(forceParam, "f", "If data already exists in the destination, the force flag will allow the target to be overwritten.")
+	// TODO: We shoould not support this for export
 	ap.SupportsFlag(contOnErrParam, "", "Continue exporting when row export errors are encountered.")
```

Comment:
@andrew-wm-arthur I don't think we should support this. It doesn't make much sense for export and was never originally tested.

Reply:
we can drop it